services/configuration/src/tests/mod.rs

```diff
@@ -276,14 +296,19 @@ fn test_discard_proposes_with_too_big_majority_count() {
 
 #[test]
 fn test_discard_proposes_with_too_small_majority_count() {
-    let mut testkit: TestKit = TestKit::configuration_default();
+    let mut testkit = TestKitBuilder::validator()
+        .with_validators(4)
+        .with_service(ConfigurationService {
+            config: ConfigurationServiceConfig {
+                majority_count: Some(2),
+            },
+        })
+        .create();
 
     let cfg_change_height = Height(5);
     let new_cfg = {
         let mut cfg = testkit.configuration_change_proposal();
-        let insufficient_majority_count = (&testkit.network().validators().len() / 2) as u16;
```

Comment:
this test is probably broken too?

We should test `majority_count < 2/3* .validators().len()` ?